plugin.cmake

```diff
@@ -1,3 +1,5 @@
+set( ENV{PYTHONPATH} ${CMAKE_CURRENT_LIST_DIR} )
```

Comment:
Adding this makes other cmake tests for girder plugin fail, since they prefer the minerva path to their own.

Reply:
Oh, I see. You probably see the reason I need this is for the https://github.com/Kitware/minerva/pull/468/files/1ce8f9f9d83968119215d0933908d4b104708081#diff-c6aae2354f158fa61778727be393104dR25
which will prepare my data for testing. 
So, at this moment, what I can think of is, 
1, omit this line in this file and ask the developer to set it before running cmake.
2, make a change to girder to make it actually try pickup the TEST_MODULE at root girder and all plugins directories?
3, Will set it first and then unset it after work?

Reply:
I've made a PR on Girder (https://github.com/girder/girder/pull/2535) to resolve this.  Assuming that gets merged, we can remove the `set` line and add `TEST_PYTHONPATH "${CMAKE_CURRENT_LIST_DIR}"` to the `minervaSpec` test.

Reply:
Oh, I see the PR. That's great. Thanks!